test/lib/ansible_test/_data/sanity/pylint/plugins/deprecated.py

```diff
@@ -6,6 +6,12 @@
 
 from distutils.version import LooseVersion
 
+try:
+    import semantic_version
+    HAS_SEMANTIC_VERSION = True
+except ImportError as dummy:
+    HAS_SEMANTIC_VERSION = False
+
```

Comment:
Since `semantic_version` is listed in the requirements I don't think we should be checking for it here. Just let it fail as it would for any other missing requirement.